runtime/j9vm/module.xml

```diff
@@ -64,6 +64,10 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 			<group name="jdk15">
 				<include-if condition="spec.java15"/>
 			</group>
+			<group name="jdk15only">
+				<include-if condition="spec.java15"/>
+				<exclude-if condition="spec.java16"/>
```

Comment:
This isn't ideal, we'll have to remember to add java17, etc.

Reply:
No, `spec.java16` means the same as `JAVA_SPEC_VERSION >= 16`.